src/vehicle.cpp

```diff
@@ -1340,28 +1297,30 @@ const vpart_info& vehicle::part_info (int index, bool include_removed) const
 
 // engines & alternators all have power.
 // engines provide, whilst alternators consume.
-int vehicle::part_power(int const index, bool const at_full_hp) const
+int vehicle::part_power(int const index, bool at_full_hp) const
 {
-    if( !part_flag(index, VPFLAG_ENGINE) &&
-        !part_flag(index, VPFLAG_ALTERNATOR) ) {
-       return 0; // not an engine.
-    }
+    const vehicle_part &vp = parts[index];
+    int pwr = 0;
 
-    const vehicle_part& vp = parts[ index ];
+    ///\EFFECT_STR increases power output of manual engines
+    if( part_flag( index, "MUSCLE_LEGS" ) ) {
+        // average man (STR 10) = 1/3 hp
+        pwr = g->u.str_cur * 0.033 * 745.7;
```

Comment:
You could use engine power for the base, though scaling needs another number.

It can't be a pure linear conversion. 10 strength isn't 10 times as strong as 1 strength, it's more like 3-4 times as strong as 0 strength, if that.